test/parallel/test-timers-unrefd-interval-still-fires.js

```diff
@@ -2,19 +2,24 @@
 /*
  * This test is a regression test for joyent/node#8900.
  */
-require('../common');
-var assert = require('assert');
+const common = require('../common');
+const assert = require('assert');
 
-var N = 5;
+const TEST_DURATION = common.platformTimeout(100);
+const N = 5;
 var nbIntervalFired = 0;
-var timer = setInterval(function() {
+
+const keepOpen = setTimeout(() => {
+  console.error('[FAIL] Interval fired %d/%d times.', nbIntervalFired, N);
+  throw new Error('Test timed out. keepOpen was not canceled');
+}, TEST_DURATION);
+
+const timer = setInterval(() => {
   ++nbIntervalFired;
-  if (nbIntervalFired === N)
+  if (nbIntervalFired === N) {
     clearInterval(timer);
+    clearTimeout(keepOpen);
```

Comment:
In here, can you set `timer._onTimeout` to a function that throws an error if it called again?